simcase-game/src/components/Statement.js

```diff
@@ -0,0 +1,32 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+
+const Container = {
+  textAlign: "center",
+  backgroundColor: "rgb(213,245,245)",
+  marginLeft: "auto",
+  marginRight: "auto",
+  width: "90%",
+  borderRadius: "4px",
+  padding: "1px 1em",
+  boxSizing: "border-box",
+  marginBottom: "16px",
+  fontSize: "1.2em"
+};
+const textStyles = {
+  color: "green",
+  fontFamily: "Arial"
```

Comment:
Not all systems include this font. Also a fallback font is required, likes `sans-serif`